README.md

```diff
@@ -265,6 +265,47 @@ import { compressors } from 'hyparquet-compressors'
 const data = await parquetReadObjects({ file, compressors })
 ```
 
+## GeoParquet Support
+
+Hyparquet reads [GeoParquet](https://github.com/opengeospatial/geoparquet) files, and can coonvert into [GeoJSON](https://datatracker.ietf.org/doc/html/rfc7946).
+
+### Why GeoParquet?
+
+GeoParquet is an emerging standard for storing geospatial vector data in the columnar [Parquet](https://parquet.apache.org/) format. By leveraging Parquet’s columnar storage, GeoParquet files can be significantly more compact and faster to query than their GeoJSON equivalents, especially for large datasets. This can result in:
+
+- **Smaller file sizes:** A GeoParquet file plus this lightweight library can often be much smaller than the equivalent GeoJSON, reducing bandwidth costs and improving load times.
+- **Faster loading:** Pure JavaScript parsing of Parquet files in the browser, with tiny package size means fast page load times.
+- **Seamless integration:** GeoParquet is based on well-established standards (GeoJSON, Parquet), ensuring broad compatibility and extensibility.
```

Comment:
I don't think we need to justify geoparquet in the hyparquet readme.

Reply:
Agreed, I'll make it shorter.